fusor/fusor.py

```diff
@@ -732,23 +738,23 @@ def add_gene_descriptor(self, fusion: Fusion) -> Fusion:
         if fusion.type == FusionType.CATEGORICAL_FUSION:
             properties.append(fusion.critical_functional_domains)
 
-        for property in properties:
-            for obj in property:
-                if "gene_descriptor" in obj.__fields__.keys():
+        for prop in properties:
```

Comment:
`property` is a decorator or some sort of keyword in python, so I changed this here to get rid of a warning